fastlane_core/lib/fastlane_core/configuration/configuration_file.rb

```diff
@@ -43,12 +43,14 @@ def initialize(config, path, block_for_missing, skip_printing_values = false)
 
       begin
         # rubocop:disable Security/Eval
-        eval(content) # this is okay in this case
+        puts(eval(content)) # this is okay in this case
```

Comment:
could we remove this `puts`?

Reply:
You may be right.
I had thought that `puts` is required to catch SyntaxError execption during `eval` … but I tried it now, it wasn’t necessary.